### PR TITLE
[MRG] MAINT: move cell_response to separate file

### PR DESCRIPTION
**Pull request by jasmainak**
I think this will simplify the organization and help with future re-organization

### CODE REVIEW

File: hnn_core/cell_response.py [hunk 1]
```diff
@@ -0,0 +1,395 @@
+"""CellResponse class."""
+
+# Authors: Nick Tolley <nicholas_tolley@brown.edu>
+#          Ryan Thorpe <ryan_thorpe@brown.edu>
+#          Mainak Jas <mjas@mgh.harvard.edu>
```
Comment (ntolley) [inline hnn_core/cell_response.py:5]: Surprised I noticed this, do you want to use the same email across files?

**Review by ntolley — APPROVED** [review]
Looks great! Glad to see `network.py` lose a couple pounds.

File: hnn_core/cell_response.py (hunk 1 above)
Comment (jasmainak) [reply hnn_core/cell_response.py:5]: I'll do this in a separate PR!

**Comment by jasmainak** [comment]
@ntolley merge button is yours!

**Pull request merged**